content/influxdb/cloud-dedicated/admin/cluster-data.md

```diff
@@ -0,0 +1,398 @@
+---
+title: Query cluster system data
```

Comment:
When I first see this, I take it to mean _Query cluster_ system data. 🤷

Reply:
Ah, ok. I can see that. What about just "Query system data"?

Reply:
```suggestion
title: Query system data
```